meshtastic/util.py

```diff
@@ -225,8 +225,7 @@ def waitForAckNak(
 
     def waitForTraceRoute(self, waitFactor, acknowledgment, attr="receivedTraceRoute") -> bool:
         """Block until traceroute response is received. Returns True if traceroute response has been received."""
-        self.expireTimeout *= waitFactor
-        self.reset()
+        self.reset(self.expireTimout * waitFactor)
```

Comment:
typo here (possibly copied from my comment on the issue, heh) in `expireTimeout`